subprojects/core/src/integTest/groovy/org/gradle/operations/configuration/IsolatedProjectsSettingsBuildOperationsIntegrationTest.groovy

```diff
@@ -18,13 +18,20 @@ package org.gradle.operations.configuration
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.BuildOperationsFixture
+import org.gradle.integtests.fixtures.ToBeFixedForIsolatedProjects
 import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
 import org.gradle.internal.configurationcache.options.ConfigurationCacheSettingsFinalizedProgressDetails
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.IntegTestPreconditions
 
 class IsolatedProjectsSettingsBuildOperationsIntegrationTest extends AbstractIntegrationSpec {
 
     def operations = new BuildOperationsFixture(executer, temporaryFolder)
 
+    @Requires(
+        value = IntegTestPreconditions.NotIsolatedProjects,
+        reason = "Exercises disabled IP behavior"
```

Comment:
❓ What is the disabled behavior?

Reply:
Bad wording apparently, sorry. It's better to say that the test is literally controls enablement of IP by itself.

Reply:
I would suggest moving the annotation to the class level.

In general, I would suspect test classes starting with `IsolatedProjects*` are either using a corresponding fixture to control the feature, or expect it to not be enabled.

Reply:
Agree, moved to the top class level